components/ServerCard.tsx

```diff
@@ -46,11 +38,14 @@ const ServerCard = () => {
         >
           {server.approval_required ? (
             <FormattedMessage
-              id="request_access"
-              defaultMessage="Request access"
+              id="servers.get_on_waitlist"
+              defaultMessage="Get on waitlist"
             />
           ) : (
-            <FormattedMessage id="join_server" defaultMessage="Join server" />
+            <FormattedMessage
+              id="servers.join_server"
```

Comment:
do we have a good way to rename the `join_server` id to `servers.join_server` and transfer all of the strings with it? there's a lot of copy on the /apps page, too, that didn't really change, but i also want to namespace the id it to the page / section it's used in

Reply:
oo should have mentioned this with #28, you can run `yarn extract` and it will update `en.json` with all the new IDs and strings, so you shouldn't have to update `en.json` manually anymore

Reply:
that makes sense for generating en.json. does that discard all of the existing translations we have in other languages if we rename a translation key though?

Reply:
good point!! just updated the `recoverTranslations` function to check for matching default messages, instead of just matching keys. so now you should be able to run `yarn extract` to grab all the new default messages and then `node recoverTranslations.js` to grab any existing translations .

Reply:
omg! mika MVP 🏆